pkg/virt-operator/resource/apply/core_test.go

```diff
@@ -395,24 +396,19 @@ var _ = Describe("Apply", func() {
 
 				Expect(hasImmutableFieldChanged(targetService, cachedService)).To(BeFalse())
 				ops, err := generateServicePatch(cachedService, targetService)
-				Expect(err).ToNot(HaveOccurred())
-
-				hasSubstring := func(ops []string, substring string) bool {
-					for _, op := range ops {
-						if strings.Contains(op, substring) {
-							return true
-						}
-					}
-					return false
+				if !expectLabelsAnnotationsPatch && !expectSpecPatch {
+					Expect(ops).To(BeEmpty())
+				} else {
 				}
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
I'd expect this to be the first instruction after `ops, err := generateServicePatch(cachedService, targetService)`